app/assets/javascripts/templates/course/discussion/post.jst.ejs

```diff
@@ -1,15 +0,0 @@
-<% formActionPath = Routes.course_topic_post_path(courseId, topicId, postId) %>
```

Comment:
Is this template used in comments center ?

Reply:
yeah, i've added it back 😛